doc/tutorials/pre-launched-rt.rst

```diff
@@ -30,60 +30,60 @@ Example of Pre-Launched RT
 
 Take the Whiskey Lake WHL-IPC-I5 board (as described in :ref:`hardware`) for
```

Comment:
WHL-IPC-I5 isn't listed in the Supported Hardware doc. However, we do have predefined files for it.